docs/src/index.md

```diff
@@ -181,8 +185,13 @@ linear (generic function with 2 methods)
 julia> grads = gradient(() -> sum(linear(x)), Params([W, b]))
 Grads(...)
 
+# Apply gradients to model parameters
 julia> grads[W], grads[b]
 ([0.652543 … 0.683588], [1.0, 1.0])
 ```
+To inspect the `Grads(...)` object returned for implicit parameters, you can index it using the parameters passed to `Params`:
+
+```julia
+julia> [grads[p] for p in [W, b]]
```

Comment:
```suggestion
```

To inspect the `Grads(...)` object returned for implicit parameters, you can index it using the parameters passed to `Params`:
```julia
julia> grads[W], grads[b]
([0.652543 … 0.683588], [1.0, 1.0])
```
```

Reply:
Ooof Github does not like all the backticks here. Hopefully, you get the idea behind the suggestion.

Reply:
I don't know how you tricked it to accept the backticks, but well done. 

I'm still not sure we should break this code block to insert the sentence. The others are kept together, and it means you can copy-paste them whole to the REPL. Can't whatever sentence of clarification come afterwards, like
```
julia> grads[W], grads[b]
([0.652543 … 0.683588], [1.0, 1.0])
` ` `
Here `grads` is a dictionary-like object, whose keys are the same parameters we 
indicated in `Params`. These parameters `W, b` are global variables, but gradients 
with respect to other global variables are not stored. 

However, implicit parameters exist mainly for compatibility with Flux...
```

Reply:
Yeah, let's keep the example together and move the text to after. I am updating my comment below.

Also, I discovered that you need to add extra backticks to the outer most block (so quandruple-backtick block instead of triple-backtick).

Reply:
I am not sure the last part about globals is correct (if I understood the sentence properly). `grads.grads` will contain extra gradients including globals that are not part of `Params`.

Reply:
Oh nice, good to know. 

And, perhaps clearer not to mention globals.